internal/winapi/filesystem.go

```diff
@@ -8,6 +8,7 @@ package winapi
 
 //sys NtOpenDirectoryObject(handle *uintptr, accessMask uint32, oa *ObjectAttributes) (status uint32) = ntdll.NtOpenDirectoryObject
 //sys NtQueryDirectoryObject(handle uintptr, buffer *byte, length uint32, singleEntry bool, restartScan bool, context *uint32, returnLength *uint32)(status uint32) = ntdll.NtQueryDirectoryObject
+//sys NtFsControlFile(file windows.Handle, event windows.Handle, apcRoutine uintptr, apcCtx uintptr, iosb *IOStatusBlock, fsControlCode uint32, in []byte, inLength uint32, out []byte, outLength uint32) (status uint32) = ntdll.NtFsControlFile
```

Comment:
```suggestion
//sys NtFsControlFile(file windows.Handle, event windows.Handle, apcRoutine uintptr, apcCtx uintptr, iosb *IOStatusBlock, fsControlCode uint32, in []byte,, out []byte) (status uint32) = ntdll.NtFsControlFile
```

The generated code automatically adds the slice length after the slice pointer, so this causes duplicates in the syscall parameters:
https://github.com/microsoft/hcsshim/blob/35ee6201cc6fd3f65e20ad0eebcee05603708a88/internal/winapi/zsyscall_windows.go#L877